src/packages/shared-types/authority.ts

```diff
@@ -1,14 +1,14 @@
-// TODO: Refactor out
-export enum Authority {
-  MED_SPA = "medicaid spa",
-  CHIP_SPA = "chip spa",
-  "1915b" = "1915(b)",
-  "1915c" = "1915(c)",
-}
+export const CHIP_SPA = "CHIP SPA" as const;
+export const MEDICAD_SPA = "Medicaid SPA" as const;
+export const WAIVER_1915_B = "1915(b)" as const;
+export const WAIVER_1915_C = "1915(c)" as const;
 
-/* Authority values as a union string type. Replaces old Authority enum. */
-export type AuthorityUnion =
-  | "Medicaid SPA"
-  | "CHIP SPA"
-  | "1915(b)"
-  | "1915(c)";
+export const AUTHORITY = {
+  [CHIP_SPA]: "chip spa",
+  [MEDICAD_SPA]: "medicaid spa",
+  [WAIVER_1915_B]: "1915(b)",
+  [WAIVER_1915_C]: "1915(c)",
+} as const;
+
+export type Authority = keyof typeof AUTHORITY;
+export type SeatoolAuthority = (typeof AUTHORITY)[keyof typeof AUTHORITY];
```

Comment:
@mdial89f `SeatoolAuthority` refers to the values of `Authority`. Would you say that's a good name?